features/aanschrijfwijze.feature

```diff
@@ -589,9 +589,9 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
         - er is geen aanspreekvorm als:
           - de persoon geen adellijke titel/predikaat heeft
           - de persoon gebruikt aanduiding naamgebruik "P" (naam partner)
-          - er is geen adellijke titel/predikaat vorm die hoort bij de geslachtsaanduiding van de persoon
-          - de persoon heeft een predikaat, heeft geslachtsaanduiding "V" (vrouw) en heeft een actuele (niet ontbonden) partner
-          - de persoon heeft een predikaat, heeft geslachtsaanduiding "V" (vrouw), heeft geen actuele (maar mogelijk wel ontbonden) partner en heeft aanduiding naamgebruik ongelijk aan "E" (eigen naam) 
+          - er geen adellijke titel/predikaat vorm is die hoort bij de geslachtsaanduiding van de persoon
+          - de persoon een predikaat heeft, de geslachtsaanduiding "V" (vrouw) en een actuele (niet ontbonden) partner
```

Comment:
moet "heeft" dan niet naar het einde van de zin? Of nog twee keer herhaald?

Reply:
aangepast